src/skmultiflow/meta/classifier_chains.py

```diff
@@ -263,10 +268,11 @@ def P(y, x, cc, payoff=np.prod):
     xy = np.zeros(D + L)
     xy[0:D] = x.copy()
     for j in range(L):
-        P_j = cc.ensemble[j].predict_proba(xy[0:D+j].reshape(1, -1))[0]  # e.g., [0.9, 0.1] wrt 0, 1
-        xy[D+j] = y[j]                                           # e.g., 1
+        P_j = cc.ensemble[j].predict_proba(
+            xy[0:D + j].reshape(1, -1))[0]  # e.g., [0.9, 0.1] wrt 0, 1
+        xy[D + j] = y[j]                                           # e.g., 1
         p[j] = P_j[y[j]]                                         # e.g., 0.1
-                                                                 # or, y[j] = 0 is predicted with probability p[j] = 0.9
+        # or, y[j] = 0 is predicted with probability p[j] = 0.9
```

Comment:
Just a minor detail, I am not sure of its importance. Is it possible to align the comments (as they were before) without breaking the length limit?

Reply:
Nice catch! I fixed it!